### PR TITLE
explain geo_point mapping in geoip-processor

### PR DESCRIPTION
**Pull request by talevy**
simple docs change to add missing mapping explanation. Users may not be aware this is 
a prerequisite for doing geo-queries on this enriched data.

### CODE REVIEW

File: docs/plugins/ingest-geoip.asciidoc [hunk 1]
```diff
@@ -180,6 +180,33 @@ Which returns:
...
+          "properties": {
+            "location": { "type": "geo_point" }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
```
Comment (rjernst) [inline docs/plugins/ingest-geoip.asciidoc]: Can you make the example code a request to set mappings or create an index so this is tested?

**Review by rjernst — COMMENTED** [review]
Looks fine, but I have one request.

File: docs/plugins/ingest-geoip.asciidoc (hunk 1 above)
Comment (talevy) [reply docs/plugins/ingest-geoip.asciidoc]: you make a reasonable request — will update.

**Comment by danielmitterdorfer** [comment]
@talevy do you still want to merge this PR?

**Comment by talevy** [comment]
thanks for bubbling this back up @danielmitterdorfer. I've consolified the snippet!

**Comment by danielmitterdorfer** [comment]
Thanks for iterating @talevy! The docs CI build failed with:

```
21:37:40 Execution failed for task ':docs:integTestCluster#wait'.
21:37:40 > java.lang.reflect.UndeclaredThrowableException (no error message)
```

I'd assume it is unrelated to your change but could you please check?

**Comment by talevy** [comment]
test this please

**Comment by talevy** [comment]
test this please

File: docs/plugins/ingest-geoip.asciidoc [hunk 2]
```diff
@@ -168,18 +170,128 @@ Which returns:
...
 // TESTRESPONSE
 
+[[mappings-note]]
+===== Recognizing Location as a Geopoint
+Although this plugin enriches your document with a `location` field containing
+the estimated latitude and longitude of the ip-address, this field will not be
+indexed as a <<geo-point,`geo_point`>> type in Elasticsearch without explicitely defining it
+as such in the mapping.
+
+For example, here would be the mapping for the example index used in the above examples
```
Comment (danielmitterdorfer) [inline docs/plugins/ingest-geoip.asciidoc]: How about being more direct? 
```suggestion
You can use the following mapping for the example index above:
```

File: docs/plugins/ingest-geoip.asciidoc [hunk 3]
```diff
@@ -168,18 +170,128 @@ Which returns:
...
+    "ip" : "80.231.5.0"
   }
 }
 --------------------------------------------------
 // TESTRESPONSE
 
+[[mappings-note]]
+===== Recognizing Location as a Geopoint
+Although this plugin enriches your document with a `location` field containing
+the estimated latitude and longitude of the ip-address, this field will not be
```
Comment (danielmitterdorfer) [inline docs/plugins/ingest-geoip.asciidoc]: "ip-address" -> "IP address"

**Review by danielmitterdorfer — APPROVED** [review]
I left a few suggestions but apart from that LGTM. Feel free to merge without another review round.

**Comment by talevy** [comment]
thank you for the suggestions @danielmitterdorfer, I've incorporated them in! Will make CI 
happy and 🚢

**Comment by talevy** [comment]
test this please

**Comment by talevy** [comment]
thanks for pushing me to get this over, was a slow little PR

**Comment by danielmitterdorfer** [comment]
But finishing just feels good. :) Thanks for pursuing @talevy.

**Pull request merged**